cpp/src/arrow/array-test.cc

```diff
@@ -1287,7 +1288,7 @@ TEST_F(TestFWBinaryArray, Builder) {
 
   std::shared_ptr<Array> result;
 
-  auto CheckResult = [&length, &is_valid, &raw_data, byte_width](const Array& result) {
+  auto CheckResult = [&length, &is_valid, &raw_data](const Array& result) {
```

Comment:
TIL you can reference local `constexpr` values inside lambdas without declaring in the capture clause